dlink-client/dlink-client-base/src/main/java/com/dlink/utils/FlinkBaseUtil.java

```diff
@@ -115,8 +117,31 @@ public static String getFlinkDDL(Table table, String tableName, FlinkCDCConfig c
         return sb.toString();
     }
 
-    public static String getSinkConfigurationString(Table table, FlinkCDCConfig config, String sinkSchemaName, String sinkTableName, String pkList) {
-        String configurationString = SqlUtil.replaceAllParam(config.getSinkConfigurationString(), "schemaName", sinkSchemaName);
+    public static String getCreateCatalogStatement(FlinkCDCConfig config) {
+        String catalogName = config.getSink().get("catalog.name");
+        List<String> catalogParamKeys = config.getSink().entrySet().stream().filter(item -> {
+            return item.getKey().startsWith("catalog.");
+        }).map(Map.Entry::getKey).collect(Collectors.toList());
+        StringBuilder sb = new StringBuilder("CREATE CATALOG ");
+        sb.append(catalogName);
+        sb.append(" WITH (\n");
+        catalogParamKeys.remove("catalog.name");
```

Comment:
construct sql statement with StringBuilder is too unfriendly to understand, better with string format, this condition needn't worry about performance